_config.yml

```diff
@@ -4,3 +4,4 @@ description: Minimal is a theme for GitHub Pages.
 show_downloads: true
 google_analytics:
 theme: jekyll-theme-minimal
+#color-scheme: auto-deafualt-dark
```

Comment:
Are there a typo `deafualt` according to README.md? I think this would be `default`. Renaming files are also needed